src/main/java/Entity/MessageChat.java

```diff
@@ -0,0 +1,17 @@
+package Entity;
+
+import org.springframework.cloud.gcp.data.spanner.core.mapping.PrimaryKey;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.Column;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.Table;
+
+@Table(name = "MessageChat")
+public class MessageChat {
+    
+    @PrimaryKey(keyOrder = 1)
+    @Column(name = "MessageID")
+    Long messageID;
```

Comment:
Can this be primitive `long` instead of the wrapper type `Long`?

Reply:
Yes, meant that. Sorry, changed.